sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -469,6 +476,7 @@ export class AssessmentController {
    */
   private transformToAssessmentDTO(
     assessment: StudentAssessment,
+    noaApproval?: boolean,
```

Comment:
Please add the new parameter to the comments.

Reply:
Updated